packages/moonstone/DayPicker/DayPicker.js

```diff
@@ -278,14 +156,12 @@ const DayPickerBase = class extends React.Component {
  * @ui
  * @public
  */
-const DayPicker = Pure(
-	Expandable(
-		Changeable(
-			{prop: 'selected', change: 'onSelect'},
-			Subscription(
-				{channels: ['i18n'], mapMessageToProps: (channel, {locale}) => ({locale})},
-				DayPickerBase
-			)
+
+const DayPicker = Expandable(
```

Comment:
Looks like you removed Pure. Was this intentional?

Reply:
Nope. Added back in!